lib/main.dart

```diff
@@ -29,7 +30,7 @@ class App extends StatelessWidget {
   Widget build(BuildContext context) {
     return GetMaterialApp(
       home: MaterialApp(
-        theme: ThemeData(fontFamily: 'Roboto'),
+        theme: theme,
```

Comment:
Add `theme` to `GetMaterialApp` instead of `MaterialApp`

Reply:
@sanket143, I have  updated changes.